app/javascript/sidebar-widget/sidebarUser.jsx

```diff
@@ -1,51 +1,43 @@
-import { h, Component } from 'preact';
+import { h } from 'preact';
+import { useCallback } from 'preact/hooks';
 import PropTypes from 'prop-types';
 import { userPropTypes } from '../common-prop-types';
 
-class SidebarUser extends Component {
-  constructor(props) {
-    super(props);
-    this.onClick = this.onClick.bind(this);
-  }
-
-  onClick() {
-    const { followUser, user } = this.props;
+const SidebarUser = ({ followUser, user }) => {
+  const onClick = useCallback(() => {
     followUser(user);
-  }
+  }, [followUser, user]);
```

Comment:
Refactor (blocking): I think in this case we don't want to employ `useCallback` as we won't get any performance boost from it ([Kent C Dodds has a great article about this](https://kentcdodds.com/blog/usememo-and-usecallback)) - instead we can keep it simple with `const onClick = () => followUser(user)`